javascript/ql/lib/semmle/javascript/Externs.qll

```diff
@@ -53,7 +53,7 @@ import javascript
  *
  * /**
  *  * @param {!Object} obj
- *  * @return {!Array<string>}
+ *  * @return {!Array&lt;string&gt;}
```

Comment:
These are wrapped in `<pre>...</pre>`, so are these changes necessary? Changing `<` to `&lt;` hurts readability.

Reply:
Reserved characters such as `<` and `>` still needs to be escaped using their HTML entity.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre

You can see that `@return {!Array<string>}` currently shows up as `@return {!Array}` because the browser treats `<string>` as an unknown element.
https://codeql.github.com/codeql-standard-libraries/javascript/semmle/javascript/Externs.qll/type.Externs$ExternalDecl.html

Reply:
Could we replace the `<pre>` with triple backquotes instead?

Reply:
You could, but the `*&#47;` in the next line would stop working.

Reply:
See - I have no idea what entity number 47 is - I had to look that up, that's pretty poor readability. Luckily the obvious answer is just to replace `&#47;` with `/`.

Reply:
Hmm, right, I see why the entity is used for `/`. But the `<pre>` isn't really a good solution - it doesn't appear to render correctly in VSCode (on predicate mouse-over).